### PR TITLE
Js menu routing

### PR DESCRIPTION
**Pull request by JustynaSmoliga**
I have added routing and menu. To test you should check the following routes:
localhost:3000/
localhost:3000/users
localhost:3000/courses
localhost:3000/sections
localhost:3000/gradesheets
localhost:3000/projects
localhost:3000/teamprojects
localhost:3000/teams 
localhost:3000/myprofile
localhost:3000/registration
localhost:3000/login

### CODE REVIEW

File: frontend/src/components/Menu/Menu.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import {
+  Link
+} from "react-router-dom";
+import { createStyles, Theme, makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
```
Comment (robert-czwartosz) [inline frontend/src/components/Menu/Menu.tsx:9]:
```
import { List, ListItem, ListItemIcon, ListItemText } from '@material-ui/core';
```

File: frontend/src/components/Menu/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import PeopleIcon from "@material-ui/icons/People";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+import AppsIcon from "@material-ui/icons/Apps";
+import AssignmentIcon from "@material-ui/icons/Assignment";
+import EmojiObjectsIcon from "@material-ui/icons/EmojiObjects";
+import SettingsIcon from "@material-ui/icons/Settings";
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
```
Comment (robert-czwartosz) [inline frontend/src/components/Menu/Menu.tsx:16]:
```
import {
    PeopleIcon,
    NorificationsIcon,
    ...
} from '@material-ui/icons';
```

File: frontend/src/components/Menu/Menu.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  const classes = useStyles();
+
+  const ListItemLink = ({ path, icon, text }: ListItemLinkProps) => {
+    return (
+      <ListItem button component={Link} to={path}>
+        <ListItemIcon>{icon}</ListItemIcon>
+        <ListItemText primary={text} />
+      </ListItem>
+    );
+  };
```
Comment (robert-czwartosz) [inline frontend/src/components/Menu/Menu.tsx:68]: I think definition of ListItemLink component shouldn't be inside Menu component.

**Review by robert-czwartosz — COMMENTED** [review]
Component looks awesome! :)
There are some codewise suggestions in comments.

File: frontend/src/components/Menu/Menu.tsx (hunk 1 above)
Comment (JustynaSmoliga) [reply frontend/src/components/Menu/Menu.tsx]: Done

File: frontend/src/components/Menu/Menu.tsx (hunk 3 above)
Comment (JustynaSmoliga) [reply frontend/src/components/Menu/Menu.tsx]: ok, I have created new folder for this component.

File: frontend/src/components/Menu/Menu.tsx (hunk 2 above)
Comment (JustynaSmoliga) [reply frontend/src/components/Menu/Menu.tsx]: Import in your example doesn't work. I left it like before.

**Review by skorpiom — APPROVED** [review]

**Comment by skorpiom** [comment]
Looks and works great!

**Pull request merged**